modules/openapi/src/alloy/openapi/DiscriminatedUnionMemberComponents.scala

```diff
@@ -81,28 +83,34 @@ class DiscriminatedUnionMemberComponents() extends OpenApiMapper {
 
         componentBuilder.putSchema(unionMixinName, unionMixinSchema)
 
```

Comment:
General thought: I'm not sure it's so good that open discriminated unions are handled in here. Would much rather have a single component / mapper that does the open-ness... but these JSONSchema/OpenAPI mappers don't have a very compositional nature, so doing that in multiple passes might require some more string manipulation and trying to reconstruct some state that we already have access to, here.